packages/fast-components-react-msft/src/progress/progress.stories.tsx

```diff
@@ -2,8 +2,15 @@ import { storiesOf } from "@storybook/react";
 import React from "react";
 import { Progress } from "./";
 
+const progressStyle: any = {
```

Comment:
Is this actually *any*?

Reply:
Importing these from `@microsoft/fast-components-class-name-contracts-msft` requires that I we have 
```
progress_circularSVG__control
progress_circularSVG__container
progress_circularSVG__page
```

Shouldn't all of the classnames in the contract be optional?